main.go

```diff
@@ -349,9 +353,16 @@ func runWithOptions(opts Options, out output, logger *zap.Logger) {
 		out.Fatalf("Failed while preparing the request: %v\n", err)
 	}
 
+	isStreamingCall := serializer.IsClientStreaming() || serializer.IsServerStreaming()
+
 	// Only make the request if the user hasn't specified 0 warmup.
 	if !(opts.BOpts.enabled() && opts.BOpts.WarmupRequests == 0) {
-		makeInitialRequest(out, transport, serializer, req)
+		if isStreamingCall {
+			makeInitialStreamRequest(out, transport, serializer, req)
+		} else {
+			makeInitialRequest(out, transport, serializer, req)
+		}
+
 	}
 
 	runBenchmark(out, logger, opts, resolved, benchmarkMethod{
```

Comment:
Based on the design, benchmark with streaming should be handle this:

```
-r '{"value": "message-1"}{"value": "message-2"}'
```

How does the benchmark will work here? Based on the code, benchmark will call the method `call` on the transport and only the same message will be used. How is the next message will be useed? Did I miss something?

Reply:
We are missing an additional PR (#5) here, PR#5 would address the benchmarking of the streams